weldx/welding/utility.py

```diff
@@ -0,0 +1,40 @@
+import pint
+import numpy as np
+
+from weldx import Q_
+from weldx.constants import WELDX_UNIT_REGISTRY
+from weldx.welding.groove.iso_9692_1 import IsoBaseGroove
+
+
+@WELDX_UNIT_REGISTRY.check(None, "[length]", "[length]/[time]", "[length]")
+# TODO: extend the ckeck decorator to check on output! # ret='[length]/[time]')
+def compute_welding_speed(
+    groove: IsoBaseGroove,
+    seam_length: pint.Quantity,
+    wire_feed: pint.Quantity,
+    wire_diameter=Q_(4, "mm"),
+):
+    """Computes how fast the torch has to be moved with the given seam length and feed
+    to fill the gap of the groove.
+
+    Parameters
+    ----------
+    groove: IsoBaseGroove
+        groove definition to compute welding speed for.
+    seam_length: pint.Quantity["length"]
+        length of the seam
+    wire_feed: pint.Quantity["length/time"]
+        feed of the wire
+    wire_diameter: pint.Quantity["length"]
+        diameter of welding wire
+
+    Returns
+    -------
+    speed: pint.Quantity["length/time"]
+    """
+    groove_area = groove.cross_sect_area
+    wire_area = np.pi / 4 * wire_diameter
```

Comment:
```suggestion
    wire_area = np.pi / 4 * wire_diameter**2
```
my bad

Reply:
mine too, as I didn't noticed the mistake :) BTW how do you directly propose such changes to be committed. This is really handy.

Reply:
@marscher if you mark a comment during review, there is a small button with +-. The tooltip says that you can also press ctrl+g. This generates the following markdown:
~~~
```suggestion
    $THE_NEW_CODE
```
~~~